packages/build-utils/test/unit.get-project-paths.test.ts

```diff
@@ -0,0 +1,75 @@
+import path from 'path';
+import { normalizePath } from '../src';
+import { getProjectPaths, ProjectPath } from '../src/get-project-paths';
+import { FixtureFilesystem } from './utils/fixture-filesystem';
+
+describe.each<{
+  fixturePath: string;
+  resultPaths: ProjectPath[];
+  skipPaths?: ProjectPath[];
+  readdirCalls: number;
+  hasPathCalls: number;
+}>([
+  {
+    fixturePath: '32-monorepo-highly-nested',
+    resultPaths: [],
```

Comment:
Lets update this fixture so it returns something. That way we confirm it can find 3 levels deep but not 4